ang/crmMosaico/CreateFromBaseTemplate.html

```diff
@@ -0,0 +1,28 @@
+<div id="bootstrap-theme" ng-controller="CrmMosaicoCreateFromBaseTemplate" crm-dialog="crmMosaicoCreatePopup">
+  <form name="crmMosaicoCreateForm">
+
+    <label for="new-tpl-title">{{:: ts('Title') }} <span class="crm-marker">*</span></label>
+    <input required class="form-control" id="new-tpl-title" ng-model="$ctrl.model.title">
+
+    <label for="new-tpl-category">{{:: ts('Category') }}</label>
+    <input class="form-control" id="new-tpl-category" ng-model="$ctrl.model.category" crm-entityref="{entity: 'OptionValue', api: {params: {option_group_id: 'mailing_template_category', is_active: 1}}, select: {allowClear:true, minimumInputLength: 0, placeholder: ts('Select')}}">
```

Comment:
@colemanw @monishdeb I was having an issue with the category sticking and had to change this to be $ctrl.model.category_id here instead of category

Reply:
Good catch @seamuslee001. Fixed.